app/narwos.py

```diff
@@ -112,10 +114,12 @@ def index():
 
     validated_files = os.listdir(
         PATH_TO_VALID_FILES)
+        
+    validated_files_to_show = [file for file in validated_files if file != '.gitkeep']
 
     return render_template(
         "index.html", 
-        files=validated_files,
```

Comment:
definetely, you do not want to pass two lists of files if one of them you do not show and do not need -> pass one list with excluded .gitkeep files